### PR TITLE
feat: Tags input for entering scopes

### LINKED ISSUE #546: Use text edit with tags for scopes
Currently scopes need to be comma separated in the Nango web UI. We should use tags instead, which would make for a cleaner UI and spares developers from thinking about the separator type.

### PR DESCRIPTION
**Pull request by Chakravarthy7102**
closes #546 
tag_input

### CODE REVIEW

**Comment by bastienbeurier** [comment]
Super cool that you added this 🙏🙏🙏

File: packages/webapp/src/pages/IntegrationCreate.tsx [hunk 1]
```diff
@@ -333,14 +334,13 @@ export default function IntegrationCreate() {
                                             </Tooltip>
                                         </div>
                                         <div className="mt-1">
-                                            <input
+                                            <TagsInput
```
Comment (bastienbeurier) [inline packages/webapp/src/pages/IntegrationCreate.tsx:337]: Could you please edit the tooltip help a couple lines above (327/328) from 

"Comma-separated ('scope1,scope2,scope3').\n Scopes are documented on the developer portal of the Integration Provider."

to 

"The list of scope should be found in the documentation of the external provider."

**Comment by bastienbeurier** [comment]
It looks like a check is failing on this one :/

**Comment by Chakravarthy7102** [comment]
> It looks like a check is failing on this one :/



> It looks like a check is failing on this one :/



> It looks like a check is failing on this one :/



> It looks like a check is failing on this one :/



> It looks like a check is failing on this one :/
lets goo 🚀🚀

**Pull request merged**